<?xml version="1.0"?>
<launch>


    <node name="ferris_wheel_motor_control" pkg="bread_gazebo" type="ferris_wheel_speed_to_motor_control.py"
    output="screen">
    </node>

      <node name="logger_node" pkg="bread_gazebo" type="logger.py"
    output="screen"/>
</launch>
